sdk/resources/Microsoft.Azure.Management.Resource/src/Generated/Models/OperationDisplay.cs

```diff
@@ -69,11 +66,5 @@ public OperationDisplay(string provider = default(string), string resource = def
         [JsonProperty(PropertyName = "operation")]
         public string Operation { get; set; }
 
```

Comment:
Removing a public property will be a breaking change. Is this intended?

Reply:
Thanks, reverted it.